components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -37,7 +37,9 @@
 #include "brave/components/brave_rewards/browser/extension_rewards_service_observer.h"
 #endif
 
-#if defined(OS_ANDROID)
+#if defined(OS_ANDROID) && defined(BRAVE_CHROMIUM_BUILD)
+#include "brave/browser/android/rewards/safetynet_check.h"
```

Comment:
a dep on `brave/browser` in components is a layering violation. safetynet_check can go in `components/brave_rewards/browser` instead

Reply:
done